middleware.js

```diff
@@ -92,9 +93,14 @@ module.exports = function(options) {
 
   var maxAge = options.maxAge || 0;
 
+    
   //Allow custom log function or default one
-  var log = options.log || function (key, val) {
-    console.error('  \x1B[90m%s:\x1B[0m \x1B[36m%s\x1B[0m', key, val);
+  var log = options.log || function (severity, key, val) {
+    if (typeof(console[severity]) === 'function') {
+        console[severity]('  \x1B[90m%s:\x1B[0m \x1B[36m%s\x1B[0m', key, val);
+    } else {
+        console.error('  \x1B[90m%s:\x1B[0m \x1B[36m%s\x1B[0m', key, val);
```

Comment:
On second thought; we will never hit this `else` block, should we remove the type checking (as you have already made sure that each call site has a valid function name). 😄